variables.tf

```diff
@@ -0,0 +1,24 @@
+variable "environment" {
+  description = "Name of the environment to deploy the resource"
+  type        = string
+}
+variable "application" {
+  description = "Name of the application, e.g. 'probate', 'divorce', use 'core' for platform infrastructure"
+  type        = string
+  validation {
+    condition = can(regex("^probate|^divorce|^core", var.application))
```

Comment:
as discussed this should be moved to configuration (yml file) rather than an input